metals/src/main/scala/scala/meta/internal/metals/ChosenBuildServers.scala

```diff
@@ -6,7 +6,9 @@ import java.sql.Timestamp
 import scala.meta.internal.metals.JdbcEnrichments._
 
 class ChosenBuildServers(conn: () => Connection, time: Time) {
-  def selectedServer(md5: String): Option[String] = {
+  final val explicit = "EXPLICIT"
```

Comment:
This `EXPLICIT` stuff is probably just going to be temporary as long as we treat Bloop as the defacto standard. Once that changes, this distinction can just go away.